variables.tf

```
@@ -38,9 +38,9 @@ variable "public_subnet_tags" {
   description = "Additional tags to set on the public subnets"
 }
 
-variable "stack" {
+variable "context" {
   type        = string
-  description = "The stack name for the VPC"
+  description = "The workload name for the VPC if used in a workload context, the environment if used in a shared network context."
```

This is not complete and it could also be an application stack or 3rd party. Why no settle on something like:

```suggestion
  description = "Used as part of the resource names to indicate they are created and used within a specific context"
```

---

And please move this variable so they are in alphabetical order again 😉

---

Done